lib/src/Uncertainty/Distribution/ChiSquare.cxx

```diff
@@ -185,6 +185,12 @@ Scalar ChiSquare::computeComplementaryCDF(const Point & point) const
   return DistFunc::pGamma(0.5 * nu_, 0.5 * x, true);
 }
 
+/* Compute the entropy of the distribution */
+Scalar ChiSquare::computeEntropy() const
+{
+  return 0.5 * nu_ + M_LN2 + SpecFunc::LogGamma(0.5 * nu_) + (1.0 - 0.5 * nu_) * SpecFunc::Psi(0.5 * nu_);
```

Comment:
Sign error?
```
return 0.5 * nu_ + M_LN2 + SpecFunc::LogGamma(0.5 * nu_) - (1.0 - 0.5 * nu_) * SpecFunc::Psi(0.5 * nu_);
```

Reply:
@dbarbier It all depends on which wikipedia page you read ;-). If you read this:
[https://en.wikipedia.org/wiki/Differential_entropy#Differential_entropies_for_various_distributions](url)
you have a minus sign, and if you read this:
[https://en.wikipedia.org/wiki/Chi-squared_distribution](url)
you have a plus sign!
The best way I found to check if it is correct is to compare the entropy of a distribution d with:
```
-d.computeLogPDF(d.getSample(100000000)).computeMean()[0]
```
If the two values are close enough (5 figures or more) it is probably correct. In this case, the winner is the plus version, so no error here.